test/functional/ui/float_spec.lua

```diff
@@ -295,6 +295,34 @@ describe('float window', function()
     eq(12, pos[2])
   end)
 
+
+  it('opened and can reset config see issue #18368', function()
+    local res = exec_lua([[
+      local bufnr = vim.api.nvim_create_buf(false, true)
+
+      local opts = {
+        width = 10,
+        height = 10,
+        col = 5,
+        row = 5,
+        relative = 'editor',
+        style = 'minimal'
+      }
+
+      local win_id = vim.api.nvim_open_win(bufnr, true, opts)
+
+      vim.api.nvim_win_set_config(win_id, {
+	width = 3,
+	height = 3,
+	row = 10,
+	col = 10,
```

Comment:
not sure this is an improvement. `row` and `col` is more meaningful with the information relative _what_ being completely local. #18368 is correct the _error message_ is quite confusing and could be improved tho.